NEWS.md

```diff
@@ -1,3 +1,22 @@
+# Updated contribution workflow. 
+
+The team of main-developers of t8code and contributors to t8code is getting bigger and we needed an improved workflow to manage all of our contributions.
+With the latest version of t8code we decided to use github project boards to have a good track of our development. Having a look at [t8code's Issue Landing page](https://github.com/orgs/DLR-AMR/projects/13) you will find the state of all issues concerning t8code. If you have a good solution for an Issue that has the state "ToDo" / is in the "ToDo"-column, feel free to assign yourself and work on it. It is highly appreciated. In the future we will have specialized Project-boards, where Issues to a certain topic are summarized. But at first, all Issues will occur on t8code's Issue Landing page.
+
+## What does change for me as a developer/contributer?
+We tried to minimize the additional overhead for you as much as possible. Nearly all of the steps that an Issue-card will do on the Issue-board are automated. An Issue should have the following life-cycle:
+
+1. Opening an Issue: It will automatically added to the Project and have the status "In-Box"
+2. Labeling an Issue: If you already know the priority and the workload of the issue you can give it a label according to its weight. Otherwise we will do it. The Issue will be moved to the "ToDo"-column by the t8ddy-bot. Soon we will provide labels to sort Issues into different topics. The Issue will then also occur on the specialized Issue boards.
```

Comment:
```suggestion
2. Labeling an issue: If you already know the priority and the workload of the issue, you can give it a label according to its weight. Otherwise we will do it. The issue will be moved to the "ToDo"-column by the t8ddy-bot. Soon we will provide labels to sort issues into different topics. The issue will then also occur on the specialized issue boards.
```